src/core/Akka.Remote/Endpoint.cs

```diff
@@ -627,8 +654,7 @@ private void ResendNacked()
         private void ResendAll()
         {
             ResendNacked();
-            _resendBuffer.NonAcked.ForEach(nonacked => _writer.Tell(nonacked));
-            RescheduleAutoResend();
+            _resendBuffer.NonAcked.Take(_settings.SysResendLimit).ForEach(nonacked => _writer.Tell(nonacked));
```

Comment:
AutoResend happens automatically now and doesn't need to be manually scheduled.

We are using the new `RemoteSettings.SysResendLimit` setting, which puts a roof on the number of system messages we can resend at once. This is designed to prevent flooding of the transport.